src/components/InvertedFlatList/BaseInvertedFlatList.js

```diff
@@ -2,8 +2,9 @@
 import _ from 'underscore';
 import React, {forwardRef, Component} from 'react';
 import PropTypes from 'prop-types';
-import {FlatList, View} from 'react-native';
```

Comment:
This caused a minor type console error - https://github.com/Expensify/App/issues/20591